app/_hub/kong-inc/acme/_index.md

```diff
@@ -288,6 +289,47 @@ schedules a renewal in the background and returns immediately.
 curl http://localhost:8001/acme -XPATCH
 ```
 
+## Monitoring and debugging
+
+The ACME plugin exposes several endpoints through Admin API that can be used for
+debugging and monitoring certificate creation and renewal.
+
+- **POST /acme**: start applying or renewing certificate and return the result; available parameter:
+  - **host**: the domain to create certificate
+  - **test_http_challenge_flow**: when set, only check for configuration sanity.
+- **PATCH /acme**: same as POST, but runs the process at background.
```

Comment:
```suggestion

### Update certificate

Apply or renew the certificate and return the result. Unlike `POST`, `PATCH` runs the process in the background. 

**Endpoint**

<div class="endpoint patch">/acme</div>

**Request body**

Attribute | Description
---:| ---
`host`<br>*required* | The domain where to create the certificate.
`test_http_challenge_flow`<br>*optional* | When set, only checks if the configuration is valid. Does not apply the certificate.
```

Am I correct in assuming that `PATCH` also takes the same arguments as `POST`? Also, can it apply a certificate if one doesn't exist, or does it update only?